### PR TITLE
Add requirement for Blob Storage instead of ALDS for FTE

### PR DESCRIPTION
**Pull request by blvckcoffee**
## Description

* Azure Data Lake Storage Gen2 is hierarchical. When using FTE with ADLS Gen2, batch operation errors can occur when deleting folders during blob cleanup. While the data is deleted, the file path remains. Because ADLS is hierarchical, folders need explicit deletion. This PR adds a line to FTE configuration documentation to make it clear that Azure Blob Storage should be used instead of ADLS.

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/admin/fault-tolerant-execution.md [hunk 1]
```diff
@@ -553,7 +553,9 @@ exchange.base-directories=s3://exchange-spooling-bucket-1,s3://exchange-spooling
 #### Azure Blob Storage
 
 The following example `exchange-manager.properties` configuration specifies an
-Azure Blob Storage container as the spooling storage destination.
+Azure Blob Storage container as the spooling storage destination. Note that you
+must use Azure Blob Storage, not Azure Data Lake Storage or any other storage
```
Comment (jhlodin) [inline docs/src/main/sphinx/admin/fault-tolerant-execution.md:557]: Suggest "any other hierarchical storage option" to offer a bit more detail.

Comment (jhlodin) [inline docs/src/main/sphinx/admin/fault-tolerant-execution.md:557]: Remove "note that", so just "You muse use .."

**Review by jhlodin — APPROVED** [review]
Couple quick comments, otherwise LGTM

**Review by linzebing — APPROVED** [review]

**Review by mosabua — APPROVED** [review]

**Comment by hashhar** [comment]
It would've been awesome if the commit message also included the information you have in the PR description.

(PR information isn't part of git history and necessitates jumping around IDE/git log and GitHub).

**Comment by blvckcoffee** [comment]
@hashhar Thanks for informing me. I will provide more context in future commit messages where the rationale for the change is not obvious.

**Pull request merged**